mac/KeymanEngine4Mac/KeymanEngine4MacTests/KeymanEngineTestsStaticHelperMethods.m

```diff
@@ -28,4 +28,17 @@ + (KMXFile *)getKmxFileForCipherMusicTests {
     KMXFile *kmxFile = [[KMXFile alloc] initWithFilePath:path];
     return kmxFile;
 }
+
++ (KMXFile *)getKmxFileForSilIpaTests {
+  NSString *path = [[NSBundle bundleForClass:[KeymanEngineTestsStaticHelperMethods class]] pathForResource:@"sil_ipa.kmx" ofType:nil];
```

Comment:
I've added sil_ipa.kmx even though the final test is narrowed down to a single-rule keyboard...